mathesar/views.py

```diff
@@ -127,7 +128,6 @@ def schema_home(request, db_name, schema_id, **kwargs):
 
 def schemas(request, db_name):
     database = get_current_database(request, db_name)
-    schema = get_current_schema(request, None, database)
```

Comment:
Since there will be no schema to be rendered, hence removed this extra query. 
NOTE: Preloading has also been removed.